### ISSUE TITLE
why are errors swallowed?

### ISSUE THREAD

**Issue by ruifortes** [dominictarr/pull-abortable#2]
shouldn't the callback be called [here](https://github.com/pull-stream/pull-abortable/blob/master/index.js#L31)

**Comment by dominictarr** [comment]
Good catch. Yes, it looks like there isn't any test coverage for the code path where the normal sink does the abort, so abort works if you call `abortable.abort()` but not if the sink does the abort.
If you can make a test case for this, I can help you fix it.
